src/plugins/intel_gpu/src/plugin/plugin.cpp

```diff
@@ -189,6 +189,9 @@ std::shared_ptr<ov::ICompiledModel> Plugin::compile_model(const std::shared_ptr<
 
     ExecutionConfig config = m_configs_map.at(device_id);
     config.set_user_property(orig_config);
+    if (context->get_engine().get_device_info().supports_immad) {
+        config.set_property(ov::intel_gpu::use_onednn(true));
+    }
```

Comment:
Do we need to set `use_onednn(true)` right before `apply_user_properties()` calls? It seems `apply_user_properties()` implements this logic too

Reply:
done